docker-compose.dev.yml

```diff
@@ -9,19 +7,27 @@ services:
       POSTGRES_PASSWORD: root
       POSTGRES_USER: postgres
       POSTGRES_DB: recipes
-    restart: always
+    ports:
+      - 5432:5432
     volumes:
       - postgres_data:/var/lib/postgresql/data
+    restart: always
 
   api_backend:
-    image: recipes:v0
-    container_name: recipes_container_v0
+    build:
+      context: ./backend
+      dockerfile: Dockerfile
```

Comment:
devowy compose powinien używać devowego dockerfile'a

Reply:
dobra bo o tym nie gadaliśmy jak ta struktura ma wyglądać - to się z czasem poprawi